catalogue/webapp/server.ts

```diff
@@ -1,22 +1,25 @@
+/* eslint-disable import/first */
 // This needs to be the first module loaded in the application
-require('@weco/common/services/apm/initApm')('catalogue-server');
+import initApm from '@weco/common/services/apm/initApm';
+initApm('catalogue-server');
```

Comment:
We should do this the same way across all apps - don't really have an opinion on the "right" way but it's different in the identity app so up to you which to change

Reply:
I'll change it in the app, and then follow up like I said in the description making things a bit better